packages/notification/src/features/sms/service.ts

```diff
@@ -46,7 +46,7 @@ export async function notifyCountryConfig(
       }),
       headers: {
         'Content-Type': 'application/json',
-        authorization: token
+        authorization: token ?? ''
```

Comment:
Is this so country config won't throw an error if the key is `undefined`? In that case it's better to handle the case in country config

Reply:
@rikukissa This endpoint has been `auth: false`  from this PR and the endpoint's handler does not use the token, it was to authenticate the user calling it. To keep the call signature of this function unchanged, as a quick hack, I did this.

Reply:
I see no reason to keep the token. Should I remove it from here?

Reply:
Yea, go for it!